c-pallets/file-bank/src/lib.rs

```diff
@@ -743,10 +753,10 @@ pub mod pallet {
 		/// Parameters:
 		/// - `file_hash`: Invalid file hash
 		#[transactional]
-		#[pallet::weight(22_777_000)]
+		#[pallet::weight(22_777_000)] // harcoded weigth?
```

Comment:
Now I rewrite a benchmark code for it to generate the corresponding weight:
https://github.com/CESSProject/cess/pull/88/files#diff-127d5bc8d5ced709e3871ca77106ae0b18c87710ea3eae9e16437a826d0d603bR354-R364